### ISSUE TITLE
Creating a node_type with scripts, scripts appearing in seemingly random order

### ISSUE THREAD

**Issue by oldManLemon** [gomorpheus/terraform-provider-morpheus#59]
So I have created a new node type but when I push it via terraform I find I have to go to the nodes and manually use the drag and drop menu in GUI to correct the order. 

The order is displayed below, simply think though 

- Script1
- script2
- script3

However when it is pushed it has a different order, it isn't even following the order in which it was created using morpheus_script_template resources. 

```terraform
resource "morpheus_node_type" "dg_kubernetes_cluster_centos7" {
  name             = "DG - Kubernetes Cluster on Centos7"
  short_name       = "kubernetes-centos7"
  technology       = "vmware"
  version          = "1.1"
  virtual_image_id = 1000 #data.morpheus_virtual_image.dop_template_ubuntu2204.id # id: 988 name: DOP-Template-Ubuntu2204
  category         = "kubernetes"

  script_template_ids = [
    morpheus_script_template.dgk8_update_rehlcentos_host.id,
    morpheus_script_template.dgk8_kubeadm_cluster_master_prep_crio.id,
    morpheus_script_template.dgk8_kubeadm_cluster_master_init.id,
    morpheus_script_template.dgk8_kubeadm_cluster_master_setup.id,
    morpheus_script_template.dgk8_kubeadm_cluster_master_secret_force.id
  ]

  service_port {
    name     = "SSH"
    port     = "22"
  }

  depends_on = [
    morpheus_script_template.dgk8_update_rehlcentos_host,
    morpheus_script_template.dgk8_kubeadm_cluster_master_prep_crio,
    morpheus_script_template.dgk8_kubeadm_cluster_master_init,
    morpheus_script_template.dgk8_kubeadm_cluster_master_secret_force,
    morpheus_script_template.dgk8_kubeadm_cluster_master_setup
  ]
}
```

**OUTPUT**

See resulting order, seems to resort to this order every time. 
![image](https://user-images.githubusercontent.com/38519887/199724884-d7f244fc-a399-486b-8609-3fe3274c8313.png)

**EXPECTED OUTPUT**

![image](https://user-images.githubusercontent.com/38519887/199725418-5636c3fb-cffe-4d42-969c-ef68f5d5c7e8.png)

**Comment by martezr** [comment]
It appears that this issue is being caused by a bug in the Morpheus API. This has been fixed and will be available in a future release of the platform. This means that the Terraform provider will be unable to properly set the order for versions of the platform prior to the version that contains the bug fix.

**Comment by oldManLemon** [comment]
Do you happen to have an an idea when that would be repaired? Otherwise thank you for the update.

**Comment by martezr** [comment]
This should be fixed in 5.4.13 and 5.5.2+ without changes to the Terraform provider.

**Comment by martezr** [comment]
This issue has been confirmed to be fixed in the versions of the Morpheus platforms mentioned above (the versions referenced are available now) without any changes to the Terraform provider.
